ros_buildfarm/templates/ci/ci_job.xml.em

```diff
@@ -241,7 +241,10 @@ parameters = [
 ))@
 @(SNIPPET(
     'archive_artifacts',
-    artifacts=['install_merged.tar.gz'],
+    artifacts=[
+      'install_merged.tar.gz',
+      'docker_colcon_build/install_list.txt',
```

Comment:
Not saying we shouldn't but could we document the utility of this?

Reply:
I added the `install_list.txt` while whiteboarding the overlay jobs. At this point, I think it would be better to re-query rosdep after downloading the merged install archive in order to satisfy runtime and downstream dependencies. It would be great to find a way to exclude the build-time dependencies taken by the underlay when running overlay jobs, but I don't think there is a straightforward way to do that right now. Could be a future optimization.

tl;dr - I'll remove the `install_list.txt` because I don't think we'll need it.